openedx/features/personalized_learner_schedules/call_to_action.py

```diff
@@ -1,25 +1,33 @@
+"""
+Creates Call to Actions for resetting a Personalized Learner Schedule for use inside of Courseware.
+"""
+
 import logging
 
 from crum import get_current_request
 
 from django.conf import settings
 from django.urls import reverse
-from django.utils.translation import gettext as _
+from django.utils.translation import ugettext as _
```

Comment:
ugettext was a python2-era workaround for python2's bytes and unicode support. It's now just an alias for `gettext`.

Reply:
Oh whoops. I misread the documentation. I thought I was switching to the good one. My bad and thanks for that catch MT!